cypress/integration/home.spec.ts

```diff
@@ -1,7 +1,5 @@
 describe('home page', () => {
   it('displays the correct text', () => {
     cy.visit('/');
-
-    cy.findByTestId('homeMessage').contains('Edit <code>src/App.tsx</code> and save to reload');
```

Comment:
Should we add expectations for this test? Currently, you just visit the home path only 🤔 
Same for the below file

Reply:
Actually, I will remove the home page file (from react template) since there's no use of it so I'll remove this test as well
and the login.spec.ts will be added on This [PR](https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/33)

Reply:
I removed this file in this https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/6f7d8911e5efec5fc7ac24ea69f8d8a49e9dda7d